ScienceJournal/Strings/ScienceJournalStrings.swift

```diff
@@ -20,6 +20,7 @@
 
 import Foundation
 
+// swiftlint:disable line_length
```

Comment:
This will need to go in the script, it is overwritten (see line 18) each time the file is generated.

Reply:
Whoops.

Reply:
I'm now realizing these changes were here because you _did_ add it to the script and then generated the file. I'm sorry 😐 

So I guess those changes should be in, as long as they came from the script generating them. I wasn't thinking

Reply:
I saw you made some changes too, so I was going to merge both together and then regenerate.